test_schemas/en/test_textfield_typeahead.json

```diff
@@ -0,0 +1,60 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Textfield Typeahead",
+    "theme": "default",
+    "description": "A questionnaire to test textfields",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "country-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "country-answer",
+                                        "label": "What is your country of birth?",
+                                        "mandatory": false,
+                                        "url": "/json/countries",
```

Comment:
I don't think it's very clear what this property is for, something like `suggestions_url` would be clearer.